pyverbs/providers/efa/efadv_enums.pxd

```diff
@@ -0,0 +1,8 @@
+# SPDX-License-Identifier: (GPL-2.0 OR Linux-OpenIB)
+
+#cython: language_level=3
+
+cdef extern from 'infiniband/efadv.h':
+
+    cpdef enum:
+        EFADV_DEVICE_ATTR_CAPS_RDMA_READ = 1 << 0
```

Comment:
remove the value assignment, it should be linked automatically.

Reply:
Done